### PR TITLE
358 android users screen

### PR DESCRIPTION
**Pull request by kerymov**

### CODE REVIEW

**Review by yev-kanivets — CHANGES_REQUESTED** [review]
Code looks good 👍 But I'm not sure how should I test it functionally, because I have a crash right on the app launch - `UsersFragment` can't access views through binding.

File: app/build.gradle [hunk 1]
```diff
@@ -74,7 +74,7 @@ dependencies {
     implementation 'com.google.firebase:firebase-auth'
     implementation 'com.google.firebase:firebase-messaging-ktx'
     // use 1.0.2 version because in 1.1.0 version WebView crashes in API 21
-    implementation 'androidx.appcompat:appcompat:1.0.2'
+    implementation 'androidx.appcompat:appcompat:1.3.1'
```
[the remark this answers is not in the document]
Comment (kerymov) [reply app/build.gradle:77]: Yes, everything is OK.

**Comment by kerymov** [comment]
It's strange that the application crashes, I don't have such problems🤔. Which device did you use for the test?

**Comment by kerymov** [comment]
Maybe the last commit will fix it?

File: app/src/main/java/com/bogdan/codeforceswatcher/features/users/UserItem.kt [hunk 2]
```diff
@@ -15,7 +15,11 @@ enum class Update { INCREASE, DECREASE, NULL }
 data class UserItem(private val user: User) {
 
     val id: Long = user.id
-    val avatarLink: String = user.avatar
+    val avatar = if (user.avatar == "https://userpic.codeforces.org/no-avatar.jpg") {
```
Comment (yev-kanivets) [inline app/src/main/java/com/bogdan/codeforceswatcher/features/users/UserItem.kt:18]: Such (computed) fields are usually moved to come after all usual fields.

**Review by yev-kanivets — APPROVED** [review]
Well, looks like I've already had this branch locally and `checkout` didn't fetch the updates as it would do for the new remote branch. So I tested some old version 🤦 Looks good now 👍 

The only problem is that I see some lags when scrolling, but probably that's because of interoperability between epoxy and Jetpack Compose. Just to be clear, the end goal is to migrate the whole screen to Jetpack Compose. Do you need a separate task for it? Or do you plan to do it when all cards are migrated to Jetpack Compose?

**Comment by kerymov** [comment]
In my opinion it's better to migrate all cards first.

**Pull request merged**